.travis.yml

```diff
@@ -0,0 +1,9 @@
+sudo: required
+dist: trusty
+language: node_js
+node_js:
```

Comment:
Can you add 7.10 as well? All Bluejay modules are supposed to support that version